app/controllers/timelineable.rb

```diff
@@ -6,13 +6,13 @@ module Timelineable
   end
 
   def timeline
-    @display_date = display_date_as_time.to_date
+    @display_date = display_date_as_time
```

Comment:
I got burned in the past assuming `@display_date` was a `Date`, so we made it so. If it can't be a `Date` how would you feel about giving it a new name?